src/UserFitness.js

```diff
@@ -0,0 +1,20 @@
+class UserFitness {
+  constructor(data, user) {
+    this.userData = this.findUser(data, user.id);
+  }
+
```

Comment:
This is an excellent way to instantiate a specific user each time a new sub-class is made. Great work on this. This, along with refactoring each sub-class is going to clean up the code so much!